public/swagger-ui/css/print.css

```diff
@@ -0,0 +1,1367 @@
+/* Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org> */
```

Comment:
I suspect that this could be handled with a Grunt task. @crhallberg might be able to provide some advice on that if you're not already familiar with the technology.